rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -380,12 +402,18 @@
                         ));
     private final CallTransaction.Function function;
     private final long cost;
+    private final Function<BlockchainConfig, Boolean> _isEnabled;
     private final BridgeMethodExecutor executor;
 
     BridgeMethods(CallTransaction.Function function, long cost, BridgeMethodExecutor executor) {
+        this(function, cost, executor, blockchainConfig -> Boolean.TRUE);
+    }
+
+    BridgeMethods(CallTransaction.Function function, long cost, BridgeMethodExecutor executor, Function<BlockchainConfig, Boolean> isEnabled) {
         this.function = function;
         this.cost = cost;
         this.executor = executor;
+        this._isEnabled = isEnabled;
```

Comment:
I don't think we use `_`ed names anywhere, I'd probably look for a different name for this.